tests/requirements-relaxed.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
This needs to be generated with Python 3.10. `nox -e pip-compile-3.10(requirements-relaxed)` will handle this automatically.